utils/openlittermap_downloader.py

```diff
@@ -0,0 +1,55 @@
+import urllib.request
+import json
+import requests
+import os
+
+
+def create_json(path, period):
+    data = requests.get(f'https://openlittermap.com/global-data?date={period}')
```

Comment:
This is the only one possibility to download the data from some specified period?